instrumentation/ratpack/ratpack-1.4/testing/src/main/groovy/io/opentelemetry/instrumentation/ratpack/server/AbstractRatpackRoutesTest.groovy

```diff
@@ -98,6 +98,7 @@ abstract class AbstractRatpackRoutesTest extends InstrumentationSpecification {
             "$SemanticAttributes.NETWORK_PROTOCOL_VERSION" "1.1"
             "$SemanticAttributes.SERVER_ADDRESS" { it == "localhost" || it == null }
             "$SemanticAttributes.SERVER_PORT" { it == app.bindPort || it == null }
+            "$SemanticAttributes.CLIENT_ADDRESS" { it == "127.0.0.1" || it == null }
```

Comment:
perhaps this should also be just `"127.0.0.1"`

Reply:
no, sometimes it will be null. similar to line 102.